src/base/ParserFaults.messages

```diff
@@ -3045,8 +3038,34 @@ cmodule: SCILLA_VERSION NUMLIT CONTRACT CID LPAREN RPAREN PROCEDURE CID LPAREN R
 ## accurate view of the past (what has been recognized so far), they
 ## may provide an INCOMPLETE view of the future (what was expected next).
 ## In state 25, spurious reduction of production scid -> CID
-## In state 72, spurious reduction of production list(targ) ->
-## In state 81, spurious reduction of production typ -> scid list(targ)
+## In state 74, spurious reduction of production list(targ) ->
+## In state 83, spurious reduction of production typ -> scid list(targ)
+##
+
+<YOUR SYNTAX ERROR MESSAGE HERE>
+
+exp_term: EMP CID CID PERIOD CID WITH
+##
+## Ends in an error in state: 50.
+##
+## t_map_value -> scid . [ TYPE TRANSITION TID THROW TARROW SPID SEND SEMICOLON RPAREN RBRACE PROCEDURE MATCH MAP LPAREN LET IN ID HEXLIT FORALL FIELD EVENT EQ EOF END DELETE CONTRACT COMMA CID BAR ARROW ACCEPT ]
+## t_map_value_allow_targs_deprecated -> scid . nonempty_list(t_map_value_args) [ TYPE TRANSITION TID THROW TARROW SPID SEND SEMICOLON RPAREN RBRACE PROCEDURE MATCH MAP LPAREN LET IN ID HEXLIT FORALL FIELD EVENT EQ EOF END DELETE CONTRACT COMMA CID BAR ARROW ACCEPT ]
+##
+## The known suffix of the stack is as follows:
+## scid
+##
+
+<YOUR SYNTAX ERROR MESSAGE HERE>
```

Comment:
We should fill in the `<YOUR SYNTAX ERROR MESSAGE HERE>`s.

Reply:
OTOH, there are plenty of other places in this file where we don't change the placeholder, so maybe we shouldn't bother - Particularly if these are really reproducible by sensible contracts.

Reply:
Reading the other error messages it appears that the standard is to replace it with a message saying:

```
Ends in an error in state: 50. Please report your example at https://github.com/Zilliqa/scilla/issues. 
```

which appears to be a way to collect examples that illustrates the syntax issue. We could update to something like this or leave it as you see fit.

Reply:
Meh I think its fine to leave it as is. Presumably if someone caught a parser error that gave them this message, they would also share the contract so I don't think knowing the error state gains us anything.